themes/default/content/docs/intro/_index.md

```diff
@@ -15,7 +15,7 @@ Pulumi is a modern infrastructure as code platform. Learn more about its most im
         <p>Learn about the Pulumi architecture, core concepts and programming model.</p>
     </div>
     <div class="w-1/2 border-solid ml-4 border-t-2 border-gray-200">
-        <h3 class="no-anchor pt-4"><a href="{{< relref "console" >}}"><i class="far fa-window-maximize pr-2"></i>Pulumi Service</a></h3>
+        <h3 class="no-anchor pt-4"><a href="{{< relref "pulumi-service" >}}"><i class="far fa-window-maximize pr-2"></i>Pulumi Service</a></h3>
```

Comment:
These should pretty much always be fully-qualified, relative to the root of the website, to avoid those awful "ambiguous reference" errors from Hugo (it's the only way Hugo can be sure it's building a link to the right place.):

```suggestion
        <h3 class="no-anchor pt-4"><a href="{{< relref "/docs/intro/pulumi-service" >}}"><i class="far fa-window-maximize pr-2"></i>Pulumi Service</a></h3>
```

Reply:
k, can fix this one here. ill come back and fix the rest on this page later on. trying to get docs unbloacked.